client/state/importer-nux/temp-selectors.js

```diff
@@ -9,6 +9,10 @@ export const getNuxUrlInputValue = state => get( state, 'importerNux.urlInputVal
 
 export const getSiteDetails = state => get( state, 'importerNux.siteDetails' );
 
+export const getSelectedImportEngine = state => get( getSiteDetails( state ), 'engine' );
+
+export const doingImportFromSiteUrl = state => get( getSiteDetails( state ), 'siteUrl' );
```

Comment:
The `doing` part of the naming here feels off, should this not be something like `getImporterSiteUrl` or similar?

Reply:
no reason -- just a poor choice of a name :)  fixed